### DISCUSSION TITLE
Allow modifying/adding own classes to a widget's DOM representation

### DISCUSSION THREAD

**Discussion by tiptronic** [pods-framework/pods#6697] [Ideas]
Currently a widget generates some static classes/ids when added to a page or post.

E.g. Pods outputs something like
`'pods_widget_list container push-bottom-medium'`
but (in order to get styled by my css-framework) I want it to be 
`'widget my-own-class content-vertical-massive'.`

Please extend the PODS widgets, so I can add my own id and classes to the widget - and remove default classes.

**Comment by quasel** [comment]
never hurts to have a filter! 
Sorry for the late reply, feel free to comment if you still have questions!

**Comment by tiptronic** [comment]
I can't even remember what I've used it for ;)
